### ISSUE TITLE
pretrained models?

### ISSUE THREAD

**Issue by sansavision** [Hammour-steak/GOUB#1]
can't find the download link that is mentioned in "Testing"

**Comment by Hammour-steak** [comment]
I add checkpoints in "here"

**Comment by zjh21** [comment]
I cannot access the checkpoints link now (26th-Feb-2025). Would you mind updating it?

**Comment by Hammour-steak** [comment]
I just tried it and even with my friends, there was no problem. I guess the VPN may have some errors and you can check it. I hope it will help you.

**Comment by zjh21** [comment]
> I just tried it and even with my friends, there was no problem. I guess the VPN may have some errors and you can check it. I hope it will help you.

Yes, I have successfully downloaded the weights. Thank you very much for checking it out for me.
